uint8 nav_status
# 01 正常状态，开始导航
# 02 正常状态，到达目标点，成功
# 03 异常状态，导航至目标点失败，放弃
# 04 状态，导航至目标点过程中暂停导航，放弃
# 05 正常状态，空闲状态下暂停导航
# 06 正常状态，04情况下继续，接近目标点，成功
# 07 正常状态，04情况下继续，未接近目标点，开始导航
# 08 正常状态，空闲状态下继续导航
# 09 异常状态，暂停状态下发送新目标点，放弃


uint8 command
# 01 继续导航
# 02 暂停导航
# 03 新目标点

string text

geometry_msgs/Pose2D pose

